router/router.go

```diff
@@ -418,18 +418,23 @@ func (worker *workerT) handleWorkerDestinationJobs() {
 	failedUserIDsMap := make(map[string]struct{})
 	for _, destinationJob := range worker.destinationJobs {
 		var attemptedToSendTheJob bool
-		payload := []byte(`{}`)
+		payload := []byte{}
 		if destinationJob.StatusCode == 200 || destinationJob.StatusCode == 0 {
 			payload = destinationJob.Message
 			if worker.canSendJobToDestination(prevRespStatusCode, failedUserIDsMap, destinationJob) {
 				diagnosisStartTime := time.Now()
+				sourceID := destinationJob.JobMetadataArray[0].SourceID
+				destinationID := destinationJob.JobMetadataArray[0].DestinationID
+				userID := ""
```

Comment:
Batched jobs and router transformed jobs can have more than 1 metadatas. So, don't increment userID.

Reply:
added comment.